docs/user-guide/odin_simulation.ipynb

```diff
@@ -0,0 +1,1049 @@
+    "                    dims=['event'],\n",
+    "                    values=sc.transpose(\n",
+    "                        sc.concat(list(velocities.values()), 'speed')\n",
+    "                    ).values,\n",
+    "                    unit='m/s',\n",
+    "                )\n",
+    "            )\n",
+    "            da.coords['sim_wavelength'] = (scc.h / scc.neutron_mass / speeds).to(\n",
+    "                unit='angstrom'\n",
+    "            )\n",
+    "\n",
+    "        return OdinSimulationRawData(da)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ess.imaging.data import get_mcstas_ob_images_path, get_mcstas_sample_images_path\n",
+    "\n",
+    "ob_file_path = FilePath(get_mcstas_ob_images_path())\n",
+    "sample_file_path = FilePath(get_mcstas_sample_images_path())\n",
+    "ob_da = load_odin_simulation_data(ob_file_path, resolution=example_resolution)\n",
+    "sample_da = load_odin_simulation_data(sample_file_path, resolution=example_resolution)\n",
+    "sample_da"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def _pixel_ids_to_x(\n",
+    "    *,\n",
+    "    pixel_id: sc.Variable,\n",
+    "    resolution: McStasManualResolution = DefaultMcStasManualResolution,\n",
+    "    detector_start_x: DetectorStartX = DefaultDetectorStartX,\n",
+    "    detector_end_x: DetectorEndX = DefaultDetectorEndX,\n",
+    ") -> sc.Variable:\n",
+    "    n_col = pixel_id % resolution[0]\n",
+    "    x_interval = (detector_end_x - detector_start_x) / resolution[0]\n",
+    "    return (\n",
+    "        detector_start_x + n_col * x_interval\n",
+    "    ) + x_interval / 2  # Center of the pixel\n",
+    "\n",
+    "\n",
+    "def _pixel_ids_to_y(\n",
+    "    *,\n",
+    "    pixel_id: sc.Variable,\n",
+    "    resolution: McStasManualResolution = DefaultMcStasManualResolution,\n",
+    "    detector_start_y: DetectorStartY = DefaultDetectorStartY,\n",
+    "    detector_end_y: DetectorEndY = DefaultDetectorEndY,\n",
+    ") -> sc.Variable:\n",
+    "    n_row = pixel_id // resolution[0]\n",
+    "    y_interval = (detector_end_y - detector_start_y) / resolution[1]\n",
+    "    return (\n",
+    "        detector_start_y + n_row * y_interval\n",
+    "    ) + y_interval / 2  # Center of the pixel\n",
+    "\n",
+    "\n",
+    "def _pixel_ids_to_position(\n",
+    "    *, x: sc.Variable, y: sc.Variable, z_pos: sc.Variable\n",
+    ") -> sc.Variable:\n",
+    "    z = sc.zeros_like(x) + z_pos\n",
+    "    var = (\n",
+    "        sc.concat([x, y, z], 'event')\n",
+    "        .fold('event', dims=['pos', 'event'], shape=[3, len(x)])\n",
+    "        .transpose(dims=['event', 'pos'])\n",
+    "        .values\n",
+    "    )\n",
+    "    return sc.vectors(dims=['event'], values=var, unit='m')\n"
+   ]
```

Comment:
I feel that most of the code at the start of this notebook should be part of a submodule in the package, and not in the notebook?

Reply:
Yeah, as Søren wanted everything before December,
I thought we don't have time to make them as a module, write test and documentation.

Also, McStas format is not stable at the moment so I didn't want to start working on a mcstas IO module...

Reply:
ok